bot/exts/holidays/holidayreact.py

```diff
@@ -83,7 +83,7 @@ class Holiday(NamedTuple):
     }
 )
 Christmas = Holiday([Month.DECEMBER], {
-    "christmas tree": Trigger(r"\b((christ|x)mas|tree)\b", ["\U0001F384"]),
+    "christmas tree": Trigger(r"\b((christ|x)mas)\b", ["\U0001F384"]),
```

Comment:
The outer group isn't needed.
```suggestion
    "christmas tree": Trigger(r"\b(christ|x)mas\b", ["\U0001F384"]),
```